core/views.py

```diff
@@ -1,32 +1,61 @@
+from django.views.generic.base import TemplateView
 import stripe
 
 from django.conf import settings
 from django.shortcuts import render
 from django.views.generic import View
+from django.contrib.auth.views import LogoutView, LoginView
+from django_registration.backends.one_step.views import RegistrationView
 
-from subscription.models import Customer, Product
+from subscription.models import Customer
 
 stripe.api_key = settings.STRIPE_SECRET_KEY
 
 
-class HomeView(View):
-    def get(self, request):
-        if not request.user.is_authenticated:
-            return render(request, 'core/home.html')
-        else:
+class Login(LoginView):
+    template_name = "core/registration/login.html"
```

Comment:
Just for the sake of managing templates in one folder we are rewriting this again. It's not ideal, use the inbuilt ones!